controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -92,16 +92,21 @@ def test_connect_to_unsupported_system(self):
         with self.assertRaises(array_errors.UnsupportedStorageVersionError):
             DS8KArrayMediator("user", "password", self.endpoint)
 
-    def test_validate_capabilities_passed(self):
-        self.array.validate_supported_capabilities(
-            {config.CAPABILITIES_SPACEEFFICIENCY: config.CAPABILITY_THIN}
+    def test_validate_space_efficiency_thin_passed(self):
+        self.array.validate_supported_space_efficiency(
+            config.SPACE_EFFICIENCY_THIN
         )
         # nothing is raised
 
-    def test_validate_capabilities_failed(self):
-        with self.assertRaises(array_errors.StorageClassCapabilityNotSupported):
-            self.array.validate_supported_capabilities(
-                {config.CAPABILITIES_SPACEEFFICIENCY: "fake"}
+    def test_validate_space_efficiency_none_passed(self):
```

Comment:
I think most of the project (hence current convention) is `_success` (instead of `_passed`)

Reply:
done